test/TritonGPU/loop-pipeline.mlir

```diff
@@ -286,13 +286,14 @@ tt.func @lut_bmm_scalar(%77: i64 {tt.divisibility=16: i32},
 // CHECK: triton_gpu.async_commit_group
```

Comment:
terminology nitpick, but can we say "indirect load" instead of "lut load" -- as the title of this PR does ?